### PR TITLE
[mpg123] Upgrade to 1.26.3

### PR DESCRIPTION
**Pull request by evpobr**
* Upgrade to 1.26
* Use CMake to build Windows port to install config-file packages:

      The package mpg123:x64-windows-static provides CMake targets:
      
      find_package(mpg123 CONFIG REQUIRED)
      target_link_libraries(main PRIVATE MPG123::libmpg123 MPG123::libout123 MPG123::libsyn123)

* Remove patches because we don't use Visual Studio project

### CODE REVIEW

File: ports/mpg123/portfile.cmake [hunk 1]
```diff
@@ -18,104 +18,35 @@ else()
...
     REPO mpg123/mpg123
     REF ${MPG123_VERSION}
     FILENAME "mpg123-${MPG123_VERSION}.tar.bz2"
     SHA512 ${MPG123_HASH}
-    PATCHES
-        0001-fix-crt-linking.patch
-        0002-fix-x86-build.patch
-        0003-add-arm-configs.patch
-        0004-add-arm64-uwp-config.patch
 )
```
Comment (NancyLi1013) [inline ports/mpg123/portfile.cmake:28]:
```suggestion
```

**Comment by NancyLi1013** [comment]
```
-- CHECK_LIB_out123_RELEASE:CHECK_LIB_out123_RELEASE-NOTFOUND
CMake Error at scripts/cmake/vcpkg_fixup_pkgconfig.cmake:232 (message):
  Library "out123" was not found! If it is a system library use the
  SYSTEM_LIBRARIES parameter for the vcpkg_fixup_pkgconfig call! Otherwise,
  correct the *.pc file
Call Stack (most recent call first):
  scripts/cmake/vcpkg_fixup_pkgconfig.cmake:298 (vcpkg_fixup_pkgconfig_check_files)
  ports/mpg123/portfile.cmake:49 (vcpkg_fixup_pkgconfig)
  scripts/ports.cmake:135 (include)
```
Could you please try to fix this problem?

File: ports/mpg123/portfile.cmake [hunk 2]
```diff
@@ -18,104 +18,34 @@ else()
...
-        ${SOURCE_PATH}/ports/MSVC++/mpg123.h
-        ${SOURCE_PATH}/src/libmpg123/fmt123.h
-        ${SOURCE_PATH}/src/libmpg123/mpg123.h.in
-        DESTINATION ${CURRENT_PACKAGES_DIR}/include
-    )
+    file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
+    file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/share)
+
+    vcpkg_copy_pdbs()
+    vcpkg_fixup_pkgconfig()
```
Comment (NancyLi1013) [inline ports/mpg123/portfile.cmake:48]: ` vcpkg_fixup_pkgconfig()` is only required on Windows?

Comment (evpobr) [reply ports/mpg123/portfile.cmake:48]: CMake build is for Windows only.

**Comment by evpobr** [comment]
@NancyLi1013: the are bugs in mpg123's CMake project, waiting for new bugfix release.

**Pull request closed without merging**